interface-definitions/load-balancing-haproxy.xml.in

```diff
@@ -0,0 +1,200 @@
+              <node name="tls">
+                <properties>
+                  <help>Transport Layer Security (TLS) options</help>
+                </properties>
+                <children>
+                  <leafNode name="ssl-bind-ciphers">
+                    <properties>
+                      <help>SSL default bind ciphers</help>
+                      <completionHelp>
+                        <list>ecdhe-ecdsa-aes128-gcm-sha256 ecdhe-rsa-aes128-gcm-sha256 ecdhe-ecdsa-aes256-gcm-sha384 ecdhe-rsa-aes256-gcm-sha384 ecdhe-ecdsa-chacha20-poly1305 ecdhe-rsa-chacha20-poly1305 dhe-rsa-aes128-gcm-sha256 dhe-rsa-aes256-gcm-sha384</list>
+                      </completionHelp>
+                      <valueHelp>
+                        <format>ecdhe-ecdsa-aes128-gcm-sha256</format>
+                        <description>ecdhe-ecdsa-aes128-gcm-sha256</description>
+                      </valueHelp>
+                      <valueHelp>
+                        <format>ecdhe-rsa-aes128-gcm-sha256</format>
+                        <description>ecdhe-rsa-aes128-gcm-sha256</description>
+                      </valueHelp>
+                      <valueHelp>
+                        <format>ecdhe-ecdsa-aes256-gcm-sha384</format>
+                        <description>ecdhe-ecdsa-aes256-gcm-sha384</description>
+                      </valueHelp>
+                      <valueHelp>
+                        <format>ecdhe-rsa-aes256-gcm-sha384</format>
+                        <description>ecdhe-rsa-aes256-gcm-sha384</description>
+                      </valueHelp>
+                      <valueHelp>
+                        <format>ecdhe-ecdsa-chacha20-poly1305</format>
+                        <description>ecdhe-ecdsa-chacha20-poly1305</description>
+                      </valueHelp>
+                      <valueHelp>
+                        <format>ecdhe-rsa-chacha20-poly1305</format>
+                        <description>ecdhe-rsa-chacha20-poly1305</description>
+                      </valueHelp>
+                      <valueHelp>
+                        <format>dhe-rsa-aes128-gcm-sha256</format>
+                        <description>dhe-rsa-aes128-gcm-sha256</description>
+                      </valueHelp>
+                      <valueHelp>
+                        <format>dhe-rsa-aes256-gcm-sha384</format>
+                        <description>dhe-rsa-aes256-gcm-sha384</description>
+                      </valueHelp>
+                      <constraint>
+                        <regex>(ecdhe-ecdsa-aes128-gcm-sha256|ecdhe-rsa-aes128-gcm-sha256|ecdhe-ecdsa-aes256-gcm-sha384|ecdhe-rsa-aes256-gcm-sha384|ecdhe-ecdsa-chacha20-poly1305|ecdhe-rsa-chacha20-poly1305|dhe-rsa-aes128-gcm-sha256|dhe-rsa-aes256-gcm-sha384)
+                        </regex>
+                      </constraint>
+                      <multi/>
+                    </properties>
+                  </leafNode>
+                  <leafNode name="tls-version-min">
```

Comment:
We should be good internet citizens and default this to 1.3

Reply:
Set default 1.3, done!